logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/publicuser/search/SearchPublicUsersUseCase.kt

```diff
@@ -30,7 +30,7 @@ internal class SearchPublicUsersUseCaseImpl(
         searchQuery: String,
         maxResultSize: Int?,
         searchUsersOptions: SearchUsersOptions
-    ): Flow<Result> {
+    ): Flow<SearchUsersResult> {
```

Comment:
Shouldn't we make sure we are flowing on `dispatchers.io` instead of specifying on the VM of the AR counterpart?

Reply:
Good point, added to use case